### DISCUSSION TITLE
Change color of shown directories at current path/input

### DISCUSSION THREAD

**Discussion by gavintoddchurch** [ohmyzsh/ohmyzsh#11333] [Help] [answered]
![image](https://user-images.githubusercontent.com/106285439/201156764-86bcee19-e8f5-43b3-a4de-7c8bae11933a.png)

I want to change these from red to another color. I've tried multiple themes and that doesn't change them. Can anyone help me with this?

**Comment by mcornella** [comment, marked as answer]
Those colors are controlled by either `$LSCOLORS` or `$LS_COLORS` variables, depending on your version of `ls`.

You can find out how to change these on Google. A separate process is needed need if these files are displayed as part of completion.
